ERCS/erc-draft-wallet-call-preparation-api.md

```diff
@@ -0,0 +1,200 @@
+---
```

Comment:
```suggestion
---
eip: 7836
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.